internal/pb/export/export.proto

```diff
@@ -55,7 +62,8 @@ message SignatureInfo {
   optional string verification_key_version = 3;
   // Alias with which to identify public key to be used for verification
   // Must be in character class [a-zA-Z0-9_]
-  // For cross-compatibility with Apple, use MCC (https://en.wikipedia.org/wiki/Mobile_country_code)
+  // For cross-compatibility with Apple, use MCC
```

Comment:
@gurayAlsac I thought we were dropping MCC requirements?

Reply:
Unfortunately only when iOS 13.6 comes out.

Reply:
Dunno, this was a reformatting change that g4 fix gave me on the flip side.

Reply:
Shall I drop this?